source/_integrations/matter.markdown

```diff
@@ -23,182 +23,241 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Matter integration allows you to control Matter devices on your local WiFi or Thread network.
+The Matter integration allows you to control Matter devices on your local WiFi or {% term thread %} network.
 
-Matter is [the new standard for home automation](https://en.wikipedia.org/wiki/Matter_(standard)) which has just been released. It is in the process of being adopted by the tech industry. Matter is a local protocol, device control is done without the need of any cloud. You can use a Matter compatible device with Home Assistant without having to connect to a vendor specific cloud.
+<p class='note'>
+Both the Matter standard itself and its implementation within Home Assistant are in a early stage, you may run into compatibility and/or early adopter problems, hence our mark of the integration as BETA.
+</p>
+
+# What is Matter ?
+
+Matter is [the new standard for home automation](<https://en.wikipedia.org/wiki/Matter_(standard)>) which has been released recently. It is in the process of being adopted by the tech industry. Matter is a local protocol. Device control is done without the need of any cloud. From a technical perspective, you can use a Matter compatible device with Home Assistant without having to connect to a vendor specific cloud. However, some vendors may require you to set up an account before you can enable Matter.
+
+Unlike the radio based protocols we're already familiar with in the IoT landscape, like Zigbee and Z-Wave, Matter makes use of standard IP-based communication. Matter is not a radio protocol, but a control protocol that runs on top of the existing network infrastructure, using your existing Wi-Fi/Ethernet routers and Thread.
+
+Home Assistant is a so called _controller_, it can control Matter based devices. Other examples of Matter controllers are the Google Nest speakers, Apple HomePods, and a SmartThings Station.
```

Comment:
```suggestion
Home Assistant is a so-called _controller_, it can control Matter based devices. Other examples of Matter controllers are the Google Nest speakers, Apple HomePods, and a SmartThings Station.
```